docs/CONFIG-VARS.md

```diff
@@ -109,6 +110,79 @@ subnet_ids = {
 }
 ```
 
+## IAM
+
+By default, two custom IAM Policies, and two custom IAM Roles (with Instance Profiles) are being created. If your site security protocol does not allow automatic creation of IAM resources, you can provide pre-created Roles using the following options:
+
+| <div style="width:50px">Name</div> | <div style="width:150px">Description</div> | <div style="width:50px">Type</div> | <div style="width:75px">Default</div> | <div style="width:150px">Notes</div> |
+| :--- | :--- | :--- | :--- | :--- |
+| cluster_iam_role_name | Name of pre-existing IAM Policy for the EKS cluster | string | "" | |
+| workers_iam_role_name | Name of pre-existing IAM Policy for the cluster node VMs | string | "" | |
```

Comment:
Since this var is for a role, shouldn't the description also mention IAM role vs policy?